config/crd/bases/operator.shipwright.io_shipwrightbuilds.yaml

```diff
@@ -44,7 +44,79 @@ spec:
                 type: string
             type: object
           status:
-            description: ShipwrightBuildStatus defines the observed state of Shipwright-Build
+            description: ShipwrightBuildStatus defines the observed state of ShipwrightBuild
+            properties:
+              conditions:
+                description: Conditions holds the latest available observations of
+                  a resource's current state.
+                items:
+                  description: "Condition contains details for one aspect of the current
```

Comment:
this looks like the `make generate` was ran from an OS with the line formatting / line split issue discussed in shipwright-io/builds

@adambkaplan @otaviof wdyt - should we be as "precise" wrt line formatting from `make generate` and controller-gen in this repo as we are in shipwright-io/builds ?

the resolution @jkhelil would be to run `make generate` from something like an ubuntu container

Reply:
the verify job is failing ... it says to run `make bundle`, so maybe it is not controller-gen related, though I'm not 100% sure (still somewhat new to operator sdk mechanics)

Reply:
resolved